packages/server-wallet/src/e2e-testing/scripts/start-load-node.ts

```diff
@@ -0,0 +1,128 @@
+import path from 'path';
+import util from 'util';
+
+import * as jsonfile from 'jsonfile';
+import yargs from 'yargs/yargs';
+import {hideBin} from 'yargs/helpers';
+import _ from 'lodash';
+import chalk from 'chalk';
+import ms from 'ms';
+
+import {
+  defaultTestWalletConfig,
+  overwriteConfigWithDatabaseConnection,
+  WalletConfig,
+} from '../../config';
+import {DBAdmin} from '../..';
+import {ARTIFACTS_DIR} from '../../../jest/chain-setup';
```

Comment:
This is a slightly awkard import. Why should this use the same `ARTIFACTS_DIR` as the chain tests?

Reply:
`ARTIFACTS_DIR` points to a directory that circleCI knows about so we tend to store all our logs there. Right now it looks like we define `ARTIFACTS_DIR` in a couple of test setup files, so I just happened to grab the one from chain setup.

This probably should be defined in one place and shared by the test setups.